src/codegen/wasm.zig

```diff
@@ -140,3 +179,28 @@ fn genCall(buf: *ArrayList(u8), decl: *Decl, inst: *Inst.Call) !void {
         .decl = target,
     });
 }
+
+fn genStore(buf: *ArrayList(u8), decl: *Decl, inst: *Inst.BinOp) !void {
+    const idx = decl.fn_link.wasm.?.getLocalidx(inst.lhs) orelse
+        return error.LocalDoesNotExist; // this is a developer error in the Zig compiler
```

Comment:
```suggestion
    const idx = decl.fn_link.wasm.?.getLocalidx(inst.lhs) orelse
        unreachable; // this is a developer error in the Zig compiler
```

or using the shorthand `.?` here would also make sense.

Reply:
Thanks @joachimschmidt557 Fixed this in all codepaths as well as the .alloc in the switch above.